client/client.go

```diff
@@ -1225,6 +1225,16 @@ func (c *Client) setupNode() error {
 	if node.Name == "" {
 		node.Name, _ = os.Hostname()
 	}
+	// TODO(dani): Fingerprint these to handle volumes that don't exist/have bad perms.
```

Comment:
👍

Reply:
This is gonna be handled in a follow up PR.